docs/content/docs/ops/state/checkpoints.md

```diff
@@ -175,6 +175,35 @@ above).
 $ bin/flink run -s :checkpointMetaDataPath [:runArgs]
 ```
 
+### Checkpoint/Savepoint guarantees
+Not always the snapshot can be successfully restored. 
```

Comment:
A nice introductory sentence would be good and maybe a bit more positive. :) E.g.: 
"The ability to store and restore an application's state is an essential and powerful feature of Flink. However, it also has limitations when it comes to changes in the application or Flink configuration. The following table gives an overview of capabilities and limitiations for the various types of savepoints and checkpoints."

Reply:
+1